config/afmd/globalConfig.sh

```diff
@@ -0,0 +1,140 @@
+# Start by exporting the shared drmaa libaries to the LD_LIBRARY_PATH
```

Comment:
I got this impression somewhere while reading the code. Some GLOBAL variables are still defined on this script. The fix is simple. We will come to this later.